release_images_training.yml

```diff
@@ -141,3 +141,16 @@ release_images:
       example: False
       disable_sm_tag: False
       force_release: False
+  14:
```

Comment:
Do you mind moving this to release_images_patches.yml from the outset? that way we can set this up for patching from the get-go without having to make changes later